t/api/02-iso.t

```diff
@@ -31,7 +31,6 @@ use Test::Warnings;
 use OpenQA::Test::Case;
 use OpenQA::Client;
 use Mojo::IOLoop;
-use Data::Dump;
```

Comment:
There's a lot of noise from `use` changes involving `Dumper` which seems unrelated 🤔️

Reply:
Yes, that's not directly related. I just noticed that there's **a lot** of dead imports and removed the ones i encountered.